src/upasana/coding_exam_1/ArrayAge.java

```diff
@@ -0,0 +1,48 @@
+/*
+Program 2 : 
+WAP to provide the difference between a Younger and an elder member of the family. Age is provided in the form of an array.
+
+Sample Input: 7 12 56 62 25
+
+Sample Output: 55
+ */
+
+package upasana.coding_exam_1;
+
+public class ArrayAge {
+	
+	int minAge(int[] age) {
+		int min=age[0];
+		for (int index = 0; index <age.length; index++) {
+			
+			if (min > age[index])
+				min = age[index];
+		}
+		return min;
+	}
+
+	int maxAge(int[] age) {
```

Comment:
this can be handle with single loop and you can use if conditions.

Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files